baker/algolia/configureAlgolia.ts

```diff
@@ -77,15 +81,20 @@ export const configureAlgolia = async () => {
         ...baseSettings,
         searchableAttributes: [
             "unordered(title)",
+            "unordered(excerpt)",
+            "unordered(tags)",
             "unordered(content)",
-            "unordered(_tags)",
             "unordered(authors)",
         ],
         customRanking: ["desc(importance)"],
-        attributesToSnippet: ["content:24"],
+        attributesToSnippet: ["excerpt:20", "content:20"],
         attributeForDistinct: "slug",
-        attributesForFaceting: ["searchable(_tags)", "searchable(authors)"],
-        disableExactOnAttributes: ["_tags"],
+        attributesForFaceting: [
+            "type",
+            "searchable(tags)",
+            "searchable(authors)",
+        ],
```

Comment:
never, at the moment. it's just nice to have them available in the Algolia admin to reason more easily about search (since there's no additional cost to us).
And note that `tags` and `authors` were there before, I just added `type`.

Reply:
yes that's nice to have indeed. I'm a bit unsure about the implications for performance here: there is probably a penalty during indexing but I'm not sure about the implications for search. I would park that question for now, since we don't have a performance issue at the moment, and facets will likely be a feature that will soon be user facing - so we'll have to compose with them anyway.